<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>305</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="sign_in_btn">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>220</y>
     <width>135</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>회원가입</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="go_back_btn">
   <property name="geometry">
    <rect>
     <x>115</x>
     <y>260</y>
     <width>135</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>돌아가기</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pw_box">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>55</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Korean" country="SouthKorea"/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string extracomment="pwpw"/>
   </property>
  </widget>
  <widget class="QLabel" name="id_label">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>20</y>
     <width>64</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ID : </string>
   </property>
  </widget>
  <widget class="QLabel" name="pw_label">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>55</y>
     <width>64</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PW : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_box">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string extracomment="id"/>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>125</y>
     <width>64</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>NAME : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_box">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>125</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string extracomment="이름"/>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_box">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Korean" country="SouthKorea"/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhEmailCharactersOnly</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string extracomment="test@mail.com"/>
   </property>
  </widget>
  <widget class="QLabel" name="email_label">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>160</y>
     <width>64</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>EMAIL : </string>
   </property>
  </widget>
  <widget class="QLabel" name="pw_check_label">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>90</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PW_CHECK : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pw_check_box">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="locale">
    <locale language="Korean" country="SouthKorea"/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string extracomment="pwpw"/>
   </property>
  </widget>
  <widget class="QLabel" name="pw_condition_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>영어 소문자 숫자 
8글자 이상 </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="result">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>285</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>id_box</tabstop>
  <tabstop>pw_box</tabstop>
  <tabstop>pw_check_box</tabstop>
  <tabstop>name_box</tabstop>
  <tabstop>email_box</tabstop>
  <tabstop>sign_in_btn</tabstop>
  <tabstop>go_back_btn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sign_in_btn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>signup_confirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>go_back_btn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>go_login()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>start()</signal>
  <signal>signal1()</signal>
  <slot>go_login()</slot>
  <slot>signup_confirm()</slot>
  <slot>id_check()</slot>
  <slot>test()</slot>
 </slots>
</ui>
